src/tools/r2rdump/NativeReader.cs

```diff
@@ -107,7 +107,7 @@ public static int ReadBits(byte[] image, int numBits, ref int bitOffset)
             int bits = bitOffset % BITS_PER_BYTE;
             int val = image[start] >> bits;
             bits += numBits;
-            if (bits > BITS_PER_BYTE)
+            while (bits > BITS_PER_BYTE)
```

Comment:
Nit: Could you adjust the comment for this method - The `start` paramref doesn't exist for this method